src/pkg/projects/Microsoft.Net.UWPCoreRuntimeSdk/Microsoft.Net.UWPCoreRuntimeSdk.builds

```diff
@@ -3,7 +3,7 @@
   <Import Project="dir.props" />
 
   <ItemGroup>
-    <Project Include="Microsoft.Net.UWPCoreRuntimeSdk.pkgproj" />
+    <Project Include="Microsoft.Net.UWPCoreRuntimeSdk.pkgproj" Condition="'$(Platform)' != 'arm64'"/>
```

Comment:
Yes we should. The additional condition would include some check == Windows_NT, correct?